code/game/objects/items/storage/belt.dm

```diff
@@ -34,10 +34,10 @@
 	item_state = "utility"
 	content_overlays = TRUE
 
-/obj/item/storage/belt/ulility/ComponentInitialize()
+/obj/item/storage/belt/utility/ComponentInitialize()

```

Comment:
....... The heck did I not notice that. Ugh.

Reply:
dude, i'm with you